.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.13
```

Comment:
Optional: Is it easy to test with multiple Go versions, as [we do with Travis](https://github.com/grailbio/bigslice/blob/eb0b542252b2e2d7ead34b9c2678a2d921684570/.travis.yml#L3-L4)?

Optional: Do we plan to migrate other repositories to use Github Actions, too? It's not especially troublesome to use different systems, but if one is better than the other we may as well reap the benefits in all.

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix
Example: https://github.com/grailbio/bazel-toolchain/blob/master/.github/workflows/ci.yml#L26

I have migrated my bazel ones to use Github Actions. It's slightly less stable, but better integrated, and at feature parity with Travis and CircleCI. I am guessing the stability will improve over time, and the time spent on integration work could be a win in itself.

FWIW, Cos [switched](https://github.com/vanadium/core/commit/694a147f5dfd7ebc2d2e5a4fb3c4fe448c7a377c) from Travis to CircleCI for Vanadium.

Reply:
More direct [example](https://github.com/actions/setup-go#usage) of testing with multiple Go versions.

Reply:
I took the approach of adding the near-minimum useful thing to this repository, as its starting point is no CI at all.

I was also using this as my experiment/beachhead with GitHub Actions, but I'm happy to lean on @siddharthab's experience. Would you just go for it and move over `bigslice` and `bigmachine`? We don't do super-sophisticated things, so I think it boils down to these basic things like stability and speed.